src/Microsoft.ServiceFabric.Actors/Runtime/ActorConcurrencyLock.cs

```diff
@@ -200,39 +200,39 @@ internal int Test_CurrentCount
             }
             catch
             {
-                // dirty handler threw and exception, release the turn lock
-                // and throw the exception back
+                // Dirty handler threw and exception, release the turn lock
+                // and throw the exception back.
                 this.turnLock.Release();
                 throw;
             }
 
-            // release the reentrancy lock but continue to hold the turn lock and release 
-            // it through ReleaseContext method after this call invocation on the actor is completed
+            // Releases the reentrancy lock but continue to hold the turn lock and release 
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.